components/cas-inbound-authenticator/src/main/java/org/wso2/carbon/identity/sso/cas/processor/SSOLogoutProcessor.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.sso.cas.processor;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.wso2.carbon.identity.application.authentication.framework.exception.FrameworkException;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.FrameworkLogoutResponse;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityMessageContext;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityProcessor;
+import org.wso2.carbon.identity.application.authentication.framework.inbound.IdentityRequest;
+import org.wso2.carbon.identity.core.util.IdentityUtil;
+import org.wso2.carbon.identity.sso.cas.constants.CASConstants;
+import org.wso2.carbon.identity.sso.cas.context.CASMessageContext;
+import org.wso2.carbon.identity.sso.cas.request.CASLogoutRequest;
+
+import java.util.HashMap;
+
+/**
+ *  Logout request processor.
+ */
+public class SSOLogoutProcessor extends IdentityProcessor {
+    private static Log log = LogFactory.getLog(SSOLogoutProcessor.class);
+
+    /**
+     * Get the name of the processor.
+     *
+     * @return the name of processor
+     */
+    @Override
+    public String getName() {
+
+        return CASConstants.CAS_CONFIG_NAME;
+    }
+
+    /**
+     * Get the priority of the processor.
+     *
+     * @return the priority
+     */
+    @Override
+    public int getPriority() {
+
+        return 4;
+    }
+
+    /**
+     * Get the callback path.
+     *
+     * @param context        the message context
+     * @return the server URL
+     */
+    @Override
+    public String getCallbackPath(IdentityMessageContext context) {
+
+        return IdentityUtil.getServerURL("/authenticationendpoint/samlsso_logout.do", false, false);
```

Comment:
Any reason to use this samlsso_logout.do here?

Reply:
Modified accordingly